extensions/amp-base-carousel/1.0/arrow.js

```diff
@@ -25,44 +25,36 @@ import {dict} from '../../../src/utils/object';
 export function Arrow(props) {
   const {
     'customArrow': customArrow,
-    'dir': dir,
+    'by': by,
```

Comment:
Should we go ahead and a type + arg destructuring?

Reply:
I can do this once #29303 is merged as it contains the type definitions.